src/vmm/src/io_uring/queue/completion.rs

```diff
@@ -93,7 +92,13 @@ impl CompletionQueue {
             self.unmasked_head += Wrapping(1u32);
             ring.store(self.unmasked_head.0, self.head_off, Ordering::Release)?;
 
-            Ok(Some(Cqe::new(cqe)))
+            let res = cqe.res;
+            #[allow(clippy::cast_possible_truncation)]
+            let index = cqe.user_data as usize;
```

Comment:
If the user data is always an index into a slab, and `slab` wants its indices to be `usize`, can we just define the type of `user_data` to be `usize` instead of `u64`?

Reply:
user_data is defined in [src/vmm/src/io_uring/bindings.rs](https://github.com/firecracker-microvm/firecracker/blob/49db07b32df5cc0dde0ff4a9e04a431f17340091/src/vmm/src/io_uring/bindings.rs#L92) which is autogenerated so changing it becomes difficult.

Reply:
Ah, bindgen strikes again, sorry for not noticing that earlier